### PR TITLE
hide unnessary fields PO and PR

### PR DESCRIPTION
**Pull request by PITPL-Gayatri-Gajjjar**
- hide unnecessary field in listview in purchase order
-Grand Total (Company Currency)  in listview in purchase receipt

### CODE REVIEW

File: erpnext/buying/doctype/purchase_order/purchase_order.json [hunk 1]
```diff
@@ -1,1298 +1,1296 @@
...
-     }
-    ],
-    "search_fields": "status, transaction_date, supplier, grand_total",
-    "show_name_in_global_search": 1,
-    "sort_field": "modified",
-    "sort_order": "DESC",
-    "track_changes": 1
-   }
\ No newline at end of file
+ "actions": [],
```
Comment (PITPL-Kaushal-Khute) [inline erpnext/buying/doctype/purchase_order/purchase_order.json]: remove indention change from json file.

File: erpnext/stock/doctype/purchase_receipt/purchase_receipt.json [hunk 2]
```diff
@@ -1,1263 +1,1262 @@
...
-     }
-    ],
-    "search_fields": "status, posting_date, supplier",
-    "show_name_in_global_search": 1,
-    "sort_field": "modified",
-    "sort_order": "DESC",
-    "track_changes": 1
-   }
\ No newline at end of file
+ "actions": [],
```
Comment (PITPL-Kaushal-Khute) [inline erpnext/stock/doctype/purchase_receipt/purchase_receipt.json]: remove indention change from json file.

**Review by PITPL-Kaushal-Khute — CHANGES_REQUESTED** [review]
remove indention change from json file.

File: erpnext/stock/doctype/purchase_receipt/purchase_receipt.json (hunk 2 above)
Comment (PITPL-Gayatri-Gajjjar) [reply erpnext/stock/doctype/purchase_receipt/purchase_receipt.json]: Done

File: erpnext/buying/doctype/purchase_order/purchase_order.json (hunk 1 above)
Comment (PITPL-Gayatri-Gajjjar) [reply erpnext/buying/doctype/purchase_order/purchase_order.json]: Done

Comment (PITPL-Gayatri-Gajjjar) [reply erpnext/buying/doctype/purchase_order/purchase_order.json]: Done

**Pull request closed without merging**